src/i18n/fi.json

```diff
@@ -129,6 +130,8 @@
     "validation-stringLengthCounter": " merkkiä jäljellä",
     "validation-requiredInContentLanguages": "Tämä kenttä vaaditaan kaikilla valituilla kielillä",
     "validation-offerIsFreeOrHasPrice": "Maksullinen tapahtuma vaatii hinnan kaikilla valituilla kielillä",
+    "validation-atLeastOneIsTrue": "Valitse vähintään yksi toistopäivä",
+    "validation-isMoreThanOne": "Toistovälin tulee olla vähintään 1 viikko",
```

Comment:
For some reason, I cannot get this validation message visible, the dialogue just stays open? The other ones seem to work alright. Perhaps there's no suitable field on the form that would display the message?